packages/turbo-ignore/src/errors.ts

```diff
@@ -2,25 +2,27 @@ import type { NonFatalErrorKey, NonFatalErrors } from "./types";
 
 export const NON_FATAL_ERRORS: NonFatalErrors = {
   MISSING_LOCKFILE: {
-    regex:
+    regex: [
       /reading (?:yarn.lock|package-lock.json|pnpm-lock.yaml):.*?no such file or directory/i,
+    ],
     message: `turbo-ignore could not complete - no lockfile found, please commit one to your repository`,
   },
   NO_PACKAGE_MANAGER: {
-    regex:
+    regex: [
       /run failed: We did not detect an in-use package manager for your project/i,
+    ],
     message: `turbo-ignore could not complete - no package manager detected, please commit a lockfile, or set "packageManager" in your root "package.json"`,
   },
   UNREACHABLE_PARENT: {
-    regex: /failed to resolve packages to run: commit HEAD\^ does not exist/i,
+    regex: [/failed to resolve packages to run: commit HEAD\^ does not exist/i],
     message: `turbo-ignore could not complete - parent commit does not exist or is unreachable`,
   },
-  UNREACHABLE_COMMIT: {
```

Comment:
Removed this as the only time it would occur is when the comparison sha is bad, and the INVALID_COMPARISON error message is more descriptive and helpful